pybossa/jobs.py

```diff
@@ -98,6 +99,20 @@ def get_quarterly_date(now):
     return datetime.combine(execute_date, now.time())
 
 
+def get_saturday_4pm_date(now):
```

Comment:
Perhaps renaming the variable and providing a default value?

```python
def get_saturday_4pm_date(start=datetime.utcnow()):
  # ...
```

If renaming `now` to `start`, the subsequent variables would also be renamed accordingly.

Reply:
This will be addressed in future PR for setting this as background job. Thanks!